### PR TITLE
[Fix] Adding missing Array.findIndex polyfill

### PR DESCRIPTION
**Pull request by mvaligursky**

### CODE REVIEW

**Review by willeastcott — APPROVED** [review]

**Comment by willeastcott** [comment]
Is this really the right way to go? I mean, this adds a 46 line polyfill when there's only one findIndex occurrence. Can't @ellthompson just rework that call to use `for (const animation in this._animationList)`?

**Comment by willeastcott** [comment]
Maybe:

```javascript
var indexOfAnimation = -1;
this._animationList.forEach((animation, idx) => {
    if (animation.name === pathString) indexOfAnimation = idx;
}
```

File: src/polyfill/array-find-index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    
+            // 6. Repeat, while k < len
+            while (k < len) {
+                // a. Let Pk be ! ToString(k).
+                // b. Let kValue be ? Get(O, Pk).
+                // c. Let testResult be ToBoolean(? Call(predicate, T, « kValue, k, O »)).
+                // d. If testResult is true, return k.
+                var kValue = o[k];
+                if (predicate.call(thisArg, kValue, k, o)) {
+                return k;
```
Comment (Maksims) [inline src/polyfill/array-find-index.js:34]: Sorry for nit-picking, its missing extra 4 spaces for indentation.

Comment (mvaligursky) [reply src/polyfill/array-find-index.js]: nice, I missed this one! done.

**Comment by mvaligursky** [comment]
> Maybe:
> 
> ```js
> var indexOfAnimation = -1;
> this._animationList.forEach((animation, idx) => {
>     if (animation.name === pathString && indexOfAnimation === -1) indexOfAnimation = idx;
> });
> ```

I used it again. Seems useful, and the code is pretty short .. most of that are short lines and comments.

**Comment by willeastcott** [comment]
OK, fair enough.

**Pull request merged**